dev/test/document.ts

```diff
@@ -957,29 +958,29 @@ describe('set document', () => {
       firestore.doc('collectionId/documentId').set({foo: 'bar'}, 'foo');
     })
         .to.throw(
-            'Argument "options" is not a valid SetOptions. Input is not an object.');
+            'Argument "options" is not a valid set() option. Input is not an object.');
```

Comment:
This reads a bit weird.  Maybe "is not a valid set() options object." ?  though admittedly that's a bit verbose.